docs/faq.rst

```diff
@@ -334,6 +334,145 @@ After successfull login the domain part will be striped and the rest used as use
+
+  rspamc -h antispam:11334 -P mailu -f 11 fuzzy_add /mail/user\@example.com/.Spam_Learn/cur/
+
+*Issue reference:* `1438`_.
+
+How can I block emails from a domain?
+`````````````````````````````````````
+
+Via the multimap filter it is possible to block emails from a sender domain. See the `official rspamd documentation`_ for more information. A local blacklist that contains the domains to be blocked can be configured via the Rspamd overrides folder. 
+
+The following steps have to be taken to configure an additional symbol (rule) that uses the multimap filter to block emails from sender domain.
+
+1. In the overrides folder create a configuration file for the multimap filter. This configuration is included by Rspamd in the main multimap configuration file. This means you do not have to use the "multimap {}" element. Files in the /mailu/overrides/rspamd/ folder folder are mapped to /etc/rspamd/override.d.
+   Create the file /mailu/overrides/rspamd/multimap.conf with contents:
+
+   .. code-block:: bash
+
+    #override.d/multimap.conf
+    #Tip: Each setting must be closed with a semi-colon ';'.
+    local_bl_domain {
+      type = "from";
+      filter = "email:domain";
+      map = "/etc/rspamd/override.d/blacklist.inc";
+      score = 14;
+      description = "Senders domain part is on the local blacklist";
+      group = "local_bl";
+      action = "reject";
+    }
+
+   Note the "action = "reject";" line. This is a so-called pre-filter. If you omit this line, then the configured score will be added to the total score of the email message. Depending on the end-score after processing all rules, a verdict is made. To override this, you can add the action line. When this symbol (rule) is fired, then this action is immediately taken and no further processing occurs. You can use the following actions:
+  
+   * discard: drop an email message, but return success for sender (should be used merely in special cases)
+
+   * reject: reject the email message
+
+   * add header: add email header to indicate spam
+
+   * no action: allow message
+
+   * soft reject: temporarily delay message (this is used, for instance, to greylist or ratelimit messages)
+
+   To move an email message to the Junk (Spam) folder, a score of 15 can be used in combination with the action "add header".
+   The above example configuration will reject all emails send from domains that are listed in '/etc/rspamd/override.d/blacklist.inc'.
+   
+
+2. In the rspamd overrides folder create a map that contains the domains to be blocked. You can use # to add comments. 
+   Create the file /mailu/overrides/rspamd/blacklist.inc with the following contents:
+
+   .. code-block:: bash
+
+     #Blacklisted domains. All emails from these domains are blacklisted and will be rejected.
+     #This file is LIVE reloaded by rspamd. Any changes are EFFECTIVE IMMEDIATELY.
+     dummy.com
+
+3. Reload rspamd
+
+   .. code-block:: bash
+
+     docker-compose scale antispam=0
```

Comment:
This works only for the docker compose setup. Other users might use swarm or kubernetes, so I wonder how to write this in a more "general" way...

Reply:
I have no knowledge of docker swarm and kubernetes. I'll rephrase the step to something general and mention the example is for docker-compose setup.
3. Reload rspamd by stopping the Rspamd container and starting the Rspamd container again. Example for docker-compose setup:

   .. code-block:: bash

     docker-compose scale antispam=0
     docker-compose scale antispam=1